saleor/graphql/webhook/subscription_types.py

```diff
@@ -438,6 +442,26 @@ class Meta:
         description = "Event sent when order metadata is updated." + ADDED_IN_38
 
 
+class OrderBulkCreated(SubscriptionObjectType):
+    orders = ConnectionField(
```

Comment:
This adds `first` `last` etc to the `order` query.  This is something that we will not be able to use from the webhook subscription.  I believe that we should have a normal list of orders. 
Again the worst thing is that we could consume all system resources, but this should be solved by the limit for the mutation that we are going to introduce.